### ISSUE TITLE
What if command is not need to be transactionable? 

### ISSUE THREAD

**Issue by oldhowl** [AleksandreJavakhishvili/DotNetCoreArchitecture#1]
How to ignore TransactionBehaviour if I have simple IRequest? 

I have error when my command markered IRequest only

TypeLoadException: GenericArguments[0], 'Application.СQRS.Baskets.Queries.GetShoppingCartItemsCount.GetShoppingCartItemsCountQuery', on 'Application.Common.Behaviours.TransactionBehaviour`2[TRequest,TResponse]' violates the constraint of type parameter 'TRequest'.

**Comment by alexeychernykh** [comment]
You are right. You need to update the application to .NET 5. Then it will work.

.NET 5 DI has some improvement about generic type resolving
https://jimmybogard.com/constrained-open-generics-support-merged-in-net-core-di-container/
